### ISSUE TITLE
change handling of GENERAL cards

### ISSUE THREAD

**Issue by dericed** [NMAAHC/nmaahcmm#100]
If the camera card type isn't identified then the camera cards script deviates from its usual behavior and adds this interview:

https://github.com/NMAAHC/nmaahcmm/blob/76946ca4f804dcabf9761c2ee0930bfd46e897f2/camera_cards#L418-L440

OK if I remove this part and simply proceed with the overall TAR/AIP settings?

**Comment by chialinchou1** [comment]
Yes, I think we can remove "select a strategy" part, but should print to the screen that the Camera card type not identified. creating TAR/AIP....

**Comment by dericed** [comment]
I can use GENERAL as the fallback for an unidentified card. I was thinking, if the card is GENERAL, then we could find the most popular video extension in the card and just process those files.

**Comment by chialinchou1** [comment]
That sounds good. So the TAR setting will just be an option. If no video files can be found/identified, we just logged it and get human interventions...

**Comment by dericed** [comment]
Ok, here's the update https://github.com/NMAAHC/nmaahcmm/commit/220b094f196ce4606252fccbd4ebc1dbdc9db81e
